src/Service/DistanceMatrix/Request/DistanceMatrixRequest.php

```diff
@@ -504,7 +505,7 @@ public function setLanguage($language = null)
     public function buildQuery()
     {
         $locationBuilder = function (LocationInterface $location) {
-            return $location->buildQuery();
+            return $location->buildQuery() . ($location instanceof EncodedPolylineLocation ? ':' : '');
```

Comment:
Hum, can't you make it part of the `buildQuery` method of the `EncodedPolylineLocation`? This way not need for the instanceof check :)